src/Razor/src/Microsoft.VisualStudio.Editor.Razor/TelemetryReporter.cs

```diff
@@ -247,6 +254,28 @@ public IDisposable TrackLspRequest(string lspMethodName, string languageServerNa
         }));
     }
 
+    private static bool IsNumeric(object? o)
+        => o is not null && IsNumeric(o.GetType());
+       
+    private static bool IsNumeric(this Type type)
```

Comment:
I unashamedly copied from https://github.com/dotnet/runtime/blob/5b448cf7dd2689bcf8ca54dd3aa9852fd94d3e1a/src/libraries/System.Linq.Expressions/src/System/Dynamic/Utils/TypeUtils.cs#L109

Reply:
I'm going to guess TypeCode is used to circumvent any problems with syntactic sugar types, and enum.GetTypeCode() returns `Int32`